membership-common/src/main/scala/com/gu/zuora/soap/Client.scala

```diff
@@ -109,8 +109,4 @@ object Client {
   def childFilter[C <: Query, P <: Query with Identifiable](parent: P): SimpleFilter =
     SimpleFilter(parent.objectName + "Id", parent.id)
 
-  val noLogPrefix: LogPrefix = new LogPrefix {
-    override def message: String = "no-id"
-  }
```

Comment:
I moved this out to the LogPrefix object so it can be used from anywhere that it's needed, and is easier to discover.